src/main/java/CMV.java

```diff
@@ -331,8 +332,23 @@ private boolean LIC12() {
    * Computes the LIC 13 condition.
    */
   private boolean LIC13() {
-    // TODO: Implementation.
-    return false;
+    if(parameters.RADIUS2 < 0 || points.length < 5) return false;
+
+    boolean containedInCircleRad1 = false;
+    boolean containedInCircleRad2 = false;
+
+    int aPts = parameters.A_PTS;
+    int bPts = parameters.B_PTS;
+    double raduis1 = parameters.RADIUS1;
+    double raduis2 = parameters.RADIUS2;
+
+
+    for(int Pnt = 0; Pnt < points.length - aPts - bPts - 2; Pnt++){
+      if(insideCircle(points[Pnt], points[Pnt + aPts + 1], points[Pnt + aPts + bPts + 2], raduis1)) containedInCircleRad1 = true;
+      if(insideCircle(points[Pnt], points[Pnt + aPts + 1], points[Pnt + aPts + bPts + 2], raduis2)) containedInCircleRad2 = true;
```

Comment:
Not sure if this checks with circles centered at the other points. Imagine a line of points and the test circle is only placed at the outer vertex of the line, then the other end may not be in the circle, but placed at the center the points would fit. Hope that made sense.